contracts/System.sol

```diff
@@ -30,6 +30,7 @@ contract System {
   address public constant TOKEN_MANAGER_ADDR = 0x0000000000000000000000000000000000001008;
   address public constant CROSS_CHAIN_CONTRACT_ADDR = 0x0000000000000000000000000000000000002000;
 
+  uint8 constant public CROSS_STAKE_CHANNELID = 0x10;
```

Comment:
Move to the upper where defining the CHANNELID is better?

Reply:
Will that change the storage?